package.json

```diff
@@ -99,7 +101,8 @@
     "ssri": "6.0.1",
     "typesafe-actions": "3.2.1",
     "typestyle": "2.0.1",
-    "visibilityjs": "2.0.2"
+    "visibilityjs": "2.0.2",
+    "snyk": "^1.161.1"
```

Comment:
Isn't this a dev-dependency?

Reply:
So `snyk` removed it from devDependency and put it in dependency.

Reply:
But I agree, I don't see why it is needed as a dependency and it should be a dev-dependency instead.

Reply:
Changes pushed